src/api/moderation.js

```diff
@@ -94,3 +94,14 @@ export const generateTestUserSubs = childId => {
     return { err };
   }
 };
+
+export const getTableInfo = childId => {
+  try {
+    return apiAuthGet(`/mod/get-info/${childId}`).then(res => {
```

Comment:
Can't tell at a glance, are you guys coordinating on this part or is this doubling up? It looks pretty close to some of @drew-ross's changes, you might have a conflict merging this section.

Reply:
It was because Sam checked out from my branch because he wanted to see the features I worked on. I'm reverting those admin panel features anyway because Trevor felt we shouldn't need to hit the API for testing purposes. I just reverted my changes and fixed Sam's branch as well.